src/i18n/uk/body.json

```diff
@@ -1,12 +1,11 @@
 {
-    "title": "ДАСМЕТАЛ",
-    "subtitle": "Композитна Арматура",
-    "introduction" : "Що це таке? Композитна арматура - це сучасний будівельний матеріал, який поєднує у собі високоміцні полімерні композити та армуючі елементи. Вона виготовляється шляхом вплетення або вкладення в армуючі елементи полімерних матеріалів, таких як скловолокно або базальтове волокно.",
-    "keyStrengths" : "Основні переваги: Міцність та Легкість: Композитна арматура має високу міцність при невеликій вазі, що полегшує транспортування та встановлення. Корозійна стійкість: Вона абсолютно стійка до корозії та хімічної агресії, що гарантує тривалий термін служби в будь-яких умовах.",
-    "versatileConstructionApplications": "Застосування в будівництві: Використовується для армування бетонних конструкцій, фундаментів, колон, балок та інших елементів будівель. Застосовується в мостах, тунелях, та інших об'єктах інфраструктури.",
-    "secondSubtitle": "Сітки Полімерно-Композитні",
-    "overview": "Що це таке? Сітки полімерно-композитні представляють собою структурні елементи, виготовлені з полімерних матеріалів та армуючих компонентів, таких як скловолокно чи вуглецеве волокно.",
-    "advantachesAtGlanze": "Основні переваги: Легкість та Простота в Монтажі: Легкість матеріалу полегшує процес транспортування та встановлення сіток. Висока Міцність: Вони володіють високою міцністю та здатні витримувати значні навантаження.",
-    "applicationsConstructionLandscape": "Застосування в будівництві: Використовується для армування бетонних покриттів, стін, стель, теплоізоляційних систем та інших конструкцій. Застосовується в архітектурних проектах для створення міцних та легких елементів."
-    
-}
\ No newline at end of file
+  "title": "ДАСМЕТАЛ",
```

Comment:
what are the changes here?

Reply:
There have been removed unnecessary indentations